Sources/kai/AST.swift

```diff
@@ -76,6 +76,8 @@ indirect enum AstNode {
     case stmtIf(cond: AstNode, body: AstNode, AstNode?, SourceRange)
     case stmtReturn([AstNode], SourceRange)
     case stmtFor(initializer: AstNode?, cond: AstNode?, post: AstNode?, body: AstNode, SourceRange)
+    case stmtSwitch(cond: AstNode?, body: AstNode, defaultBody: AstNode?, SourceRange)
+    case stmtCase(cond: AstNode, body: AstNode, SourceRange)
```

Comment:
Not sure what to change `cond` to here, possibly `match` or something? (best to omit the label?).
Lets also have body be an array of nodes, same reason as for the `stmtSwitch`